atomate/atat/database.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+
+from __future__ import division, print_function, unicode_literals, absolute_import
+
+"""
+This module defines the database classes.
+"""
+
+from atomate.utils.database import CalcDb
+from atomate.utils.utils import get_logger
+
+__author__ = 'Matthew Horton'
+__credits__ = 'Anubhav Jain, Kiran Mathew, Josua Vieten'
+__email__ = 'mkhorton@lbl.gov'
+# based on FeffCalcDb
+
+logger = get_logger(__name__)
+
+class McsqsCalcDb(CalcDb):
```

Comment:
We should decide on whether the various names should look like McsqsCalcDb, McsqsDrone, etc. or like SQSCalcDb, SQSDrone, etc.

Happy to defer to your judgement, just think about it